src/main/webapp/app/jsp/my-events.jsp

```diff
@@ -55,6 +56,20 @@
         location.href = "<c:url value='/app/jsp/add-event.jsp?id='/>" + x;
     }
 
+    function deleteEvent(x) {
+        fetch("<c:url value='/api/event/delete/'/>" + x, {
+            "method": "POST",
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            },
+        }).then(function (response) {
+            if (response.status === 200) {
+                console.log("login success");
+                location.reload();
+            }       })
```

Comment:
Please use proper indentation. In IDEA, `Ctrl + Shift + L` can fix that for you.